mava/utils/make_env.py

```diff
@@ -147,7 +147,7 @@ def make_jaxmarl_env(
         add_global_state,
     )
 
-    train_env, eval_env = add_extra_wrappers(train_env, eval_env, config)
+    train_env, eval_env = add_extra_wrappers(train_env, eval_env, config)  # type: ignore
```

Comment:
Why do we have to `# type: ignore` here now? Would be great if we could fix this :pray:

Reply:
So this is from changing one of the mypy rules to a default rule. Honestly it's an us problem and should be an error, it's because `train_env` and `eval_env` for some reason don't conform to the `MarlEnv` spec, I'll have a quick look but it might not be easily fixable

Reply:
I tried but I couldn't find a fix for this. Added missing method so the wrappers should now comply with `MarlEnv` but mypy is complaining :shrug: